test/gtest/uct/ib/test_cqe_zipping.cc

```diff
@@ -94,7 +94,8 @@ class test_cqe_zipping : public test_uct_ib_with_specific_port {
         FOR_EACH_ENTITY(entity) {
             md = ucs_derived_of((*entity)->md(), uct_ib_mlx5_md_t);
             if (!(ucs_test_all_flags(md->flags, UCT_IB_MLX5_MD_FLAG_CQE64_ZIP |
-                                     UCT_IB_MLX5_MD_FLAG_CQE128_ZIP))) {
+                                     UCT_IB_MLX5_MD_FLAG_CQE128_ZIP |
```

Comment:
Code alignment?

Reply:
I would expect UCT_IB_MLX5_MD_FLAG_CQE64/128_ZIP would be 0 if UCT_IB_MLX5_MD_FLAG_DEVX_CQ is 0. Isn't this the case?

Reply:
BTW , why was this issue found with ODP?